mattermost-plugin/webapp/src/components/rhsChannelBoardItem.scss

```diff
@@ -38,5 +38,10 @@
         -webkit-line-clamp: 3;
         -webkit-box-orient: vertical;
         overflow: hidden;
+        cursor: pointer;
+
+        .MarkdownEditor {
```

Comment:
I believe this object `.MarkdownEditor` was removed...so this is no longer necessary.

Reply:
Okay.